src/main/java/org/jenkinsci/plugins/mesos/Mesos.java

```diff
@@ -35,12 +35,15 @@ public JenkinsSlave(String name) {
     JenkinsSlave slave;
     final double cpus;
     final int mem;
+    final String role;
     final MesosSlaveInfo slaveInfo;
 
-    public SlaveRequest(JenkinsSlave slave, double cpus, int mem, MesosSlaveInfo slaveInfo) {
+    public SlaveRequest(JenkinsSlave slave, double cpus, int mem, String role,
```

Comment:
line wrapping. can you put one argument per line? what's the standard in java world?